gspread/client.py

```diff
@@ -390,6 +390,17 @@ def remove_permission(self, file_id, permission_id):
         headers = {'Content-Type': 'application/json'}
 
         self.session.delete(url, headers=headers)
+        
+    def update_format(self, file_id, data_file):
```

Comment:
change 'data_file' to 'data'

Reply:
@TelmoM thanks for weighing in. However, I think it would be better if you can provide a more elaborated comment on the suggested change.

Reply:
Hi @TelmoM ,
yes, you are right. The argument in the update_format method should be named "data" instead of "data_file". The POST in the end uses "data" as var name for the content of the json. Sorry about this.